src/renderer/components/SimpleTransactions.vue

```diff
@@ -91,6 +100,28 @@ export default {
         color: $red;
       }
     }
+
+    &.status {
+      text-align: right;
+
+      .aph-icon {
+        svg {
+          height: toRem(14px);
+
+          &.confirmed {
+            .fill {
+              fill: $green !important;
```

Comment:
This is interesting. I'm assuming you tried other ways to style this? There some other styling override this?

Reply:
Yeah, sucks but I tried.